.github/workflows/actions/configuration/action.yaml

```diff
@@ -67,23 +67,3 @@ runs:
     run: |
       python tests/test_sample_raycluster_yamls.py
     shell: bash
-
-  - name: Run tests for sample RayJob YAML files with the nightly operator.
```

Comment:
Why do we remove this one? Has this one been moved to Buildkite?

Reply:
Let's merge this first: https://github.com/ray-project/kuberay/pull/1321 it adds it to buildkite.

Reply:
It appears this has already been removed in some other PR.